packages/ckeditor5-bookmark/src/bookmarkui.ts

```diff
@@ -339,6 +339,17 @@ export default class BookmarkUI extends Plugin {
 		} );
 	}
 
+	/**
+	 * When bookmark element is selected, form button label should be set on 'Update' otherwise it is 'Insert'.
+	 */
+	private _updateFormButtonLabel() {
```

Comment:
In the methods where we already fire this function we already have `this._getSelectedBookmarkElement()`, maybe it would be better to pass it as a parameter (or just boolean) to not have to execute this function again?

For example:

```suggestion
	private _updateFormButtonLabel( isBookmarkSelected: boolean ) {
```